doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md

```diff
@@ -58,9 +59,9 @@ java -jar /path/to/avro-tools-1.7.7.jar compile schema <schema file> <destinatio
 
 Please refer to [Compiling the schema](http://avro.apache.org/docs/current/gettingstartedjava.html#Compiling+the+schema) for more information. It is also possible to integrate the schema compilation with [avro-maven-plugin](http://avro.apache.org/docs/current/gettingstartedjava.html).
 
-## User verifier implementation
+## Owner verifier implementation
 
-All Kaa user verifiers extend generic abstract class org.kaaproject.kaa.server.common.verifier.AbstractUserVerifier<T>. The following code example illustrates the implementation of a custom user verifier.
+All Kaa owner verifiers extend generic abstract class org.kaaproject.kaa.server.common.verifier.AbstractUserVerifier<T>. The following code example illustrates the implementation of a custom owner verifier.
```

Comment:
In code, it is still AbstractKaaUserVerifier, so I can't change it now. Thre is appropriate ticket on refactoring names of this classes